themes/mlibrary_new/exhibit-builder/exhibits/tags.php

```diff
@@ -1,5 +1,6 @@
   <div class="col-sm-12 tag-cloud">
      <li>Browse by Popular Tags</li>
-  </div>
+</div>
 
-<ul class="tag-list"><?php echo tag_string($tags,'exhibits/browse');?></ul>
+<ul class="tag-list"><?php
+echo $tags = tag_string($tags,'exhibits/browse');?></ul>
```

Comment:
The `$tags` isn't actually used.

Reply:
It does, but maybe I do not need it by echo the tag_string directly. I meant to keep the results in tags then display the tags.

Reply:
Right, I didn't qualify that.  $tags in the left-hand side isn't used after the assignment.